client/source/js/modules/model/cost-coverage.html

```diff
@@ -70,8 +67,8 @@
               </p>
               <p>
                 3. [Optional] Specify scale-up parameters:
-                <input type="number" name="scaleUpParameter" min="1" max="10" step="0.1" class="txbox __inline __xl" placeholder="e.g.{{defaultScaleUpParameter}}"  ng-model="scaleUpParameter" ng-change="updateCurves()">
-                <error-messages for="scaleUpParameter" rules="{min:1, max:10}"></error-messages>
```

Comment:
@Ashalynd try something like [0.6, 0.5, 7000000.0, None, None]. The OST program is much more expensive so ccparams[2] needs to be much higher before you really see an effect